streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/VariableController.java

```diff
@@ -59,8 +60,13 @@ public class VariableController {
     @PostMapping("page")
     @RequiresPermissions("variable:view")
     public RestResponse page(RestRequest restRequest, Variable variable) {
-        IPage<Variable> variableList = variableService.page(variable, restRequest);
-        return RestResponse.success(variableList);
+        IPage<Variable> page = variableService.page(variable, restRequest);
+        for (Variable v : page.getRecords()) {
+            if (v.getDesensitization()) {
+                v.setVariableValue(ConfigConst.DEFAULT_DATAMASK_STRING());
```

Comment:
I suggest to define the method in the Variable class:

```
public void dataMasking() {
        if (desensitization) {
            String dataMask = ConfigConst.DEFAULT_DATAMASK_STRING();
            this.setVariableValue(dataMask);
        }
    }
```
In this way, we just call `dataMasking` method.

Reply:
good idea